creator-node/src/middlewares.js

```diff
@@ -266,18 +266,29 @@ async function ensureStorageMiddleware(req, res, next) {
 /**
  * Issue SyncRequests to both secondaries, and wait for at least one to sync before returning
  * @dev TODO - move out of middlewares layer
```

Comment:
is there context on why this shouldn't be a middleware?

Reply:
yeah its not called as a middleware currently, just a function that some routes call